File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = true;
    
    if(s.size()==4||s.size()==6){
        for(auto a:s){
            if(!(a>='0'&&a<='9')){
                return false;
            }
        }
        return true;
    }
    
    return false;
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        
        int len = s.length();
        
        for(int i=0;i<len;i++){
            if(Character.isAlphabetic(s.charAt(i))){
                return false;
            }
        }
        return 4==len||len==6;
    }
}